hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -479,4 +479,17 @@ message DeletedBlocksTransactionInfo {
     optional int64 containerID = 2;
     repeated int64 localID = 3;
     optional int32 count = 4;
-}
\ No newline at end of file
+}
+
+message FileInfoProto {
```

Comment:
The proto message name needs to be more specific

```suggestion
message CompactionLogFileInfoProto {
```

Reply:
Will change it to `CompactionFileInfoProto`.